autogpt/logs.py

```diff
@@ -192,9 +192,12 @@ def emit(self, record):
 
         msg = self.format(record)
         try:
-            words = msg.split()
+            words = re.split("[ \t]+", msg)
             for i, word in enumerate(words):
-                print(word, end="", flush=True)
+                if word.endswith("\n"):
+                    print(word, end="\n", flush=True)
+                else:
+                    print(word, end="", flush=True)
```

Comment:
I'm not sure why we would want this. In most cases, it's fine if `\n` is printed symbolically instead of literally.

However, I would be supportive of an implementation that prints "long text arguments" as code blocks instead of the current `{ key: value }` style. :) That should be handled at a somewhat higher level in the code, not in the `TypingConsoleHandler`.